### DISCUSSION TITLE
About the Gift Shop in the upcoming mission pass

### DISCUSSION THREAD

**Discussion by Leroy8888** [HelixNebulaStudio/RiseOfTheDead#285] [General]
### The concept

The goal when it was discussed in unof was to offer people customization rewards from prior events at the cost of a steep grind / price to justfiy their reappearance.  Not something to quickly get just because spare tokens from the ''junk'' (`coal`, `sulfur`) reward table were used.

### Feedback Gift Shop

The contents of the origins pass seem to be randomly spread out. In my opinion they all should be at least 40 tokens as they date back from quite some time. The unlockable posters should probably also be around 100 tokens instead. The `zriceraskullinferno` is also placed rather high for 20 tokens, while it appeared in the easter event a few weeks ago.

I think it would be of a better interest to make this exclusive to skins you get from post rewards but as result lower token cost from origins items. The items are currently a bit easy to grind in large masses especially with the MPbooks, which wasn't the intent of the concept. The concept was to offer these unlockables again later down the line, but at the same time make them more difficult to obtain.

I am aware that the intent of the concept discussed in the unof server might diviate from the intent of the developer, but It is important to keep in mind that event and seasonal items should always remain some sort of value to satisfy a part of the community and not forgo their goals completely or make it more difficult for them.

### Linked items

I've also been thinking of a new mechanic that makes them exclusive to the player who unlocked them. This feature would be **linked items**, which cannot be traded or sold and can only be used by the person who bought the items with tokens. This mechanic would allow the original items from the origins pass to keep a bit of their value as after the event the demand would rise again, while at the same time allowing people to grind for a cosmetic they really like(d) to have. Items which these skins are applied on should also be made **linked** for this reason.

This mechanic would be exclusive to rewards from the Gift Shop. More specifically, from prior events. It _should_ not be applied on new content from events. It could also be used on some other things in the main game. For example, safehouse gifts.

----

And lastly as a seperate question, I've noticed the addition of the `apron` in the Gift Shop, was or is this planned to be part of the continuation of the easter event questline?

**Comment by Leroy8888** [comment]
Shortened the post a bit since it really was just an essay.

**Comment by MXKhronos** [comment]
Thanks for the feedback, I'll adjust the token cost for the items soon so that it's better balanced.
